File: vs/Listas/Lista 2/soma.c
#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, "Portuguese"); // Declaração de idioma usado, para utilização de acentos.
    int x=0,y=0,i=0,soma=0;
    
    printf("Insira números naturais x e y, para estabelecer o intervalo (x,y):  \n");
    scanf("%d %d",&x,&y);
    
    i=x;
    if (x>y){
        for (i=y+1;i<x;i++){
            if (i%2!=0){
                soma=soma+i;
            }   
            else{
                soma=soma;
            }
            
        }
    }
    else if (y>x){
        for (i=x+1;i<y;i++){
            if(i%2!=0){
                soma=soma+i;
            }
        }
    }
    else if(x==y){
            if(x%2!=0){
            soma=x;
        }
    }
    
    
    if (x>y){
    	printf("\nA soma dos números ímpares contidos no intervalo (%d,%d) é igual a %d\n",y,x,soma);
	}
	else if (x<y){
		printf("\nA soma dos números ímpares contidos no intervalo (%d,%d) é igual a %d\n",x,y,soma);
	}
    else if (x==y){
    	if (soma!=0){
    		printf("\nOs números digitados são iguais e é ímpar, com o valor %d\n",soma);
		}
		else {
			printf("\nOs números digitados são iguais e não ímpar.");
		}
	}
    
    return 0;   
}

File: vs/Listas/Lista 2/soma.py
def maior():
    x=int(input("\nInsira um valor para x e y tal para estabelecer o intervalo (x,y).\nInsira o valor de x: "))
    y=int(input("Insira o valor de y: "))


    if x>y:
        print("\nO intervalo é determinado igual a: ")
        print(y , x)
        
        if y%2!=0:
            soma=y
        elif y%2==0:
            soma=y-1
        i=soma
        for i in range(x):
            if i%2!=0:
                soma=soma+i
                print(soma)
           
    elif y>x:
        
        print("\nO intervalo é determinado igual a: ")
        print(x , y)
        if x%2!=0:
            soma=x
        elif x%2==0:
            soma=x-1
        i=soma
        for i in range(y):
            if i%2!=0:
                soma=soma+i
                print(soma)
    elif x==y:
        soma=x

    print("\nO valor da soma dos números ímpares contidos no intervalo é igual a:")
    print(soma)
    
maior()
